Tesla.css

```diff
@@ -0,0 +1,477 @@
+}
+
+.section__p {
+  font-size: 15px;
+  font-weight: 500;
+  margin: 0 72px 54px 52px;
+}
+
+.section__icon {
+    margin: 0 123px 171px 48px;
+}
+
+.item__icon {
+  display: flex;
+  align-items: center;
+  margin-bottom: 52px;
+}
+
+.icon__img {
+  margin-right: 20px;
+}
+
+.icon__img_bag {
+  margin-right: 24px;
+}
+
+.icon__text_b {
+  display: block;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.icon__text_s {
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.icon_number {
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+}
+
+.link__section {
+  color: #FFF;
+  font-weight: 600;
+  font-size: 13px;
+  text-transform: uppercase;
+  margin-left: 67px;
+  margin-right: 25px;
+  display: flex;
+  align-items: center;
+}
+
+.link__section::before {
+  content: url(img/info.svg);
+  margin-right: 7px;
+}
+
+.section__info {
+  white-space: nowrap;
+  display: flex;
+  align-items: center;
+  margin-bottom: 63px;
+}
+
+.safe {
+  display: flex;
+  position: relative;
+}
+
+.section__safe {
+  width: 30%;
+}
+
+.section__safe_img {
+  width: 70%;
+}
+
+.section__h1_safe {
+  white-space: normal;
+}
+
+.section__p_safe {
+  margin-bottom: 49px;
+}
+
+.section__h2 {
+  font-weight: 600;
+  font-size: 30px;
+  margin-left: 56px;
+  margin-bottom: 26px;
+}
+
+.ul {
+  margin-left: 56px;
+  padding-left: 0;
+}
+
+.li {
+  list-style-type: none;
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 17px;
+  margin-bottom: 16px;
+}
+
+.li:last-child {
+  margin-bottom: 0;
+}
+
+.li__span {
+  font-weight: bold;
+  font-size: 10px;
+}
+
+.star {
+  width: 25px;
+  height: 25px;
+}
+
+.stars {
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+  max-width: 25px;
+  grid-column-gap: 4px;
+  grid-row-gap: 12px;
+}
+
+.section__safe_flex {
+  display: flex;
+  align-items: center;
+  margin-right: 90px;
+  margin-bottom: 100px;
+}
+
+.safe__text {
+  margin-right: 31px;
+}
+
+.five {
+  display: flex;
+  align-items: center;
+  margin-bottom: 60px;
+}
+
+.five__icon {
+  margin-right: 23px;
+  margin-left: 66px;
+}
+
+.five__p {
+  font-weight: 600;
+  font-size: 50px;
+}
+
+.five__span {
+  font-size: 25px;
+}
+
+.section__safe_img {
+  background: url(img/safe-bg.svg) 931px 269px;
+  background-repeat: no-repeat;
+  background-position: top 97px left 189px;
+}
+
+.safe__img {
+  width: 1049px;
+  height: 540px;
+  position: absolute;
+  margin-top: 15%;
+  margin-left: 24%;
+}
+
+/*.line271 {
+  margin-top: 207px;
+  margin-left: 185px;
+  position: absolute;
+  position: fixed;
+  z-index: 2;
+}*/
+
+.description__p {
+  position: absolute;
+  font-weight: 600;
+  font-size: 17px;
+}
+
+.description__p_1 {
+  margin-top: 219px;
+  margin-left: 189px;
+}
+
+.description__p_2 {
+  margin-top: 171px;
+  margin-left: 446px;
+}
+
+.description__p_3 {
+  margin-top: 677px;
+  margin-left: 520px;
+}
+
+.mobile__img {
+  display: none;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)